kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/PurchaseOrderRetransmitDocument.java

```diff
@@ -0,0 +1,68 @@
+package edu.arizona.kfs.module.purap.document;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.businessobject.Building;
+import org.kuali.rice.krad.util.ObjectUtils;
+import edu.arizona.kfs.sys.businessobject.BuildingExtension;
+
+/**
+ * Purchase Order Retransmit Document
+ */
+public class PurchaseOrderRetransmitDocument extends org.kuali.kfs.module.purap.document.PurchaseOrderRetransmitDocument {
+    protected static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PurchaseOrderRetransmitDocument.class);
+
+    protected String routeCode;
+    protected Building buildingObj;
+
+    /**
+     * Gets the routeCodeObj attribute. 
```

Comment:
Please remove or correct this comment, this is the getter for object buildingObj, not routeCodeObj.